externs/svg-externs.js

```diff
@@ -0,0 +1,10367 @@
+/**
+ * @fileoverview Generated by closureidl 0.2dev closureidl-parser-webkit 0.2dev dart-idl 7592.
```

Comment:
What is this file, and does it need to be in the committed version of blockly or can it just be generated during the build step?